### ISSUE TITLE
SQlite error

### ISSUE THREAD

**Issue by MrKittle** [WirelessRedstoneGroup/WirelessRedstone#211]
## Bug report

**I'm using version:** 4.0.0
**I'm on Minecraft:** 1.12.2
**I'm running:** Spigot

**Seen results:**  
Error while enabling WirelessRedstone 4.0.0 in console showing an SQlite issue.

**Expected results:**  
I expected that my actual yaml file was converted into a SQlite storage type.

**How to reproduce:**  
Use an older version of WR and let it generate files for the first time. Then create channels in game to fill the file with some rows. Stop your server and replace WR's Jar file with the latest version 4.0.0 and start it again.

**Additional info:**  
Console error logs : [https://paste.md-5.net/mavugiboju.md](url)

**Comment by Bart-0110** [comment]
> How to reproduce:
Use an older version of WR and let it generate files for the first time. Then create channels in game to fill the file with some rows. Stop your server and replace WR's Jar file with the latest version 4.0.0 and start it again.

Are you inserting into the database yourself or via the plugin? It seems that there are duplicate WirelessReceiverDelayer entries.

**Comment by MrKittle** [comment]
Hello, I insert nothing in the database by myself. I had a YML file storage and I changed the config to SQlite. On starting, I suppose that the plugin tried to import yaml file content to SQlite file and it générated this error.

I stopped the server and start it again and now the yaml file disappeared and there is an SQlite file instead and no error this time. All my channels are present and work well.

Do I need to test the v4.0.1-beta.1  above ?

I thank you very much for your help :-)

Cordially.
